recipes/openfoam/8/mc/environments.yaml

```diff
@@ -0,0 +1,9 @@
+openfoam-org-env:
+  compiler:
+  - toolchain: gcc
+    spec: gcc@11
+  mpi:
+    spec: cray-mpich
+  unify: true
+  specs:
+  - openfoam-org@8
```

Comment:
You can add two views:
1. a `default` view that will put openfoam and its dependencies in the environment
2. a `develop` view that excludes openfoam, for folk who want to "build their own" openfoam.

See the following for an example of how this is done:
https://github.com/eth-cscs/alps-uenv/blob/main/recipes/gromacs/2023/mc/environments.yaml#L19-L27

Reply:
See https://github.com/eth-cscs/alps-uenv/pull/60/commits/e96549f6b13f9c64404c10f440064a7e274d9a9f

Since I'm not sure that an OpenFOAM user does not need all dependencies to build their stuff upon OpenFOAM (e.g. an extension), also the "user" view is not limited to just "run" dependencies.